src/FormControl.js

```diff
@@ -46,6 +47,7 @@ class FormControl extends React.Component {
       id = controlId,
       inputRef,
       className,
+      bsSize,
```

Comment:
@jquense do you think `bsSize` is correct here, given that it's not using the `bsClass`? should this instead be `inputSize`?